### ISSUE TITLE
Tests currently failing on development branch

### ISSUE THREAD

**Issue by lenkan** [WebOfTrust/keria#152]
I am getting some type errors when running tests on the development branch. I think it is because of recent breaking changes in keripy (https://github.com/WebOfTrust/keripy/pull/633). For example

```
def test_identifier_collection_end(helpers):
        with helpers.openKeria() as (agency, agent, app, client), \
                habbing.openHby(name="p1", temp=True) as p1hby, \
                habbing.openHby(name="p2", temp=True) as p2hby:
            end = aiding.IdentifierCollectionEnd()
            resend = aiding.IdentifierResourceEnd()
            app.add_route("/identifiers", end)
            app.add_route("/identifiers/{name}", resend)
    
            groupEnd = aiding.GroupMemberCollectionEnd()
            app.add_route("/identifiers/{name}/members", groupEnd)
    
            client = testing.TestClient(app)
    
            res = client.simulate_post(path="/identifiers", body=b'{}')
            assert res.status_code == 400
            assert res.json == {'description': "required field 'icp' missing from request",
                                'title': '400 Bad Request'}
    
            salt = b'012[34](https://github.com/lenkan/keria/actions/runs/7248372481/job/19744152853#step:6:35)[56](https://github.com/lenkan/keria/actions/runs/7248372481/job/19744152853#step:6:57)789abcdef'
            serder, signers = helpers.incept(salt, "signify:aid", pidx=0)
            assert len(signers) == 1
            signer0 = signers[0]
>           diger0 = serder.digers[0]
E           AttributeError: 'SerderKERI' object has no attribute 'digers'. Did you mean: 'ndigers'?
```

The full run is here: https://github.com/lenkan/keria/actions/runs/7248372481/job/19744152853

My current workaround is to pin keripy to a commit from dec 6th. What do you think about pinning the keripy dependency to a commit? Similar to how a package-lock file works in npm.

Workflow for breaking changes to keripy could be something like this:

1. Commit keripy changes
1. Locally upgrade keripy dedendency in keria
1. Run tests, if breaking, make appropriate fixes
1. Create pull request that contains the updated dependency together with the fixes.

**Comment by psteniusubi** [comment]
I'm seeing this too. 

How about triggering workflows of keria, signifypy etc from keripy workflow?

**Comment by lenkan** [comment]
Resolved by #154 . We can create a new issue regarding pinning if needed.
